scripts/gitlab-rpc-docs.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+clone_repos() {
+    git clone https://github.com/parity-js/jsonrpc.git jsonrpc
+    git clone https://github.com/paritytech/wiki.git wiki
+}
+
+build_docs() {
+    cp parity-ethereum jsonrpc/.parity
+    npm install
+    npm run build:markdown
+}
+
+update_wiki_docs() {
+    for file in $(ls jsonrpc/docs); do
+        module_name=${file:0:-3}
+        mv jsonrpc/docs/$file wiki/JSONRPC-$module_name-module.md
+    done
+}
+
+setup_git() {
+    git config user.email "runner@parity.com"
+    git config user.name "Parity Runner"
```

Comment:
Does this user exist? Do we need to create some user that has access?

Reply:
I would doubt that user exists. I just put that there as a placeholder for now. You will need to create a user in gitlab for the runner to use. Can then update those lines with the correct email and username.

Reply:
@General-Beck please suggest what we should use here :)